test/unit/dialects/sqlite/query-generator.test.js

```
@@ -168,8 +168,12 @@ if (dialect === 'sqlite') {
           expectation: 'SELECT count(*) AS `count` FROM `foo`;',
           context: QueryGenerator
         }, {
+          arguments: ['myTable', {order: 'id; DROP TABLE myTable;'}],
+          expectation: 'SELECT * FROM `myTable` ORDER BY `id; DROP TABLE myTable;`;',
```

This is what should throw an exception instead